### ISSUE TITLE
Meteor and Starter kit reloads are very slow or don't happen at all

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction-development-platform#29]
Also, Docker can consume most/all of system resources causing the host machine to become unusable. The consensus so far seems to be that the root cause is filesharing between Docker and OSX when using file watchers.

Some research done by @spencern 

> Short story: the major issue is with hyperkit and involves how filesystem events are handled between the host and container - see https://github.com/docker/for-mac/issues/1759#issuecomment-436818181
If you look at the stats when running reaction-platform `docker stats` you’ll notice that the CPU utilization of any particular docker container isn’t out range.

> Even `reaction` in dev mode is using only 5-50% of a single core for me.
When looking at system performance using `Activity Monitor` it tells a bit of a different story.
looking at `htop` It appears that hyperkit is chewing up CPU cycles

**Comment by carlosbaraza** [comment]
Yes, developing under Docker seems to be impossible at the moment.

**Comment by spencern** [comment]
Hi @carlosbaraza 
Try applying the attached patch to your version of Reaction. 
[meteor-polling-interval.patch.zip](https://github.com/reactioncommerce/reaction-platform/files/2617605/meteor-polling-interval.patch.zip)

It adds the following line to the `reaction` docker compose file
```
METEOR_WATCH_POLLING_INTERVAL_MS: 10000
```

If you've installed via `reaction-platform` using `make` this will need to be applied inside the `/reaction` directory inside the platform installation.
```
> cd reaction
> git apply patchfile.patch
```

What this patch does is cause Meteor to poll less frequently - especially when developing on Docker using OSX, it noticeably improves performance.

**Comment by amorimlucas** [comment]
Didn't notice any change after applying this patch. I still get a combined 100%-200% CPU usage from the containers.

**Comment by andrewtch** [comment]
Any update on that or alternative way to run it on mac? @spencern ?

Same results with latest master and macos

**Comment by spencern** [comment]
@andrewtch @amorimlucas We're still trying to figure out how to improve this. 
There's a new PR https://github.com/reactioncommerce/reaction/pull/4992 that aims to improve this. Would be interested to hear feedback on whether it helps you if you have a chance to check it out.

**Comment by hammannja** [comment]
Are there any plans to extend this functionality to Windows-based Docker containers?
Or, do I need to test a dev version of reaction-platform rather than the release candidate?

Right now, no refresh is taking place with these containers in the rc10 release for Windows-based Docker containers.

Is there another way to develop in WIndows in the meantime?

**Comment by spencern** [comment]
Hi @hammannja 
We're not able to support Windows development right now, but are interested in improving the experience for developers on Windows. 
We'd recommend using the latest version (rc.11 currently), but I don't know if that will solve your problems. If you're interested in helping figure this out, we'd be happy look at any PRs or ideas related to improving Windows support.

**Comment by aldeed** [comment]
Closing as fixed in 3.0. If more specific examples of this crop up, create an issue in that project's repo
